python/lsst/pipe/tasks/calibrateImage.py

```diff
@@ -670,8 +729,15 @@ def _measure_aperture_correction(self, exposure, bright_sources):
             necessary for point source determination for the aperture correction
             calculation.
         """
-        result = self.measure_aperture_correction.run(exposure, bright_sources)
-        exposure.setApCorrMap(result.apCorrMap)
+        norm_ap_corr_map = exposure.apCorrMap
+        ap_corr_map = self.measure_aperture_correction.run(exposure, bright_sources).apCorrMap
+
+        # Need to merge the aperture correction map from the normalization.
+        if norm_ap_corr_map:
```

Comment:
I don't think this `if` is necessary, right? `exposure.apCorrMap` should always exist at the start of this method.